modules/caddyhttp/metrics.go

```diff
@@ -108,7 +109,8 @@ func newMetricsInstrumentedHandler(handler string, mh MiddlewareHandler) *metric
 func (h *metricsInstrumentedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request, next Handler) error {
 	server := serverNameFromContext(r.Context())
 	labels := prometheus.Labels{"server": server, "handler": h.handler}
-	statusLabels := prometheus.Labels{"server": server, "handler": h.handler, "method": r.Method, "code": ""}
+	method := sanitizeMethod(r.Method)
+	statusLabels := prometheus.Labels{"server": server, "handler": h.handler, "method": method, "code": ""}
```

Comment:
In hindsight, I wonder if we should add a comment as to why "code" is `""` (empty string) -- that is significant, right? It looks like a mistake at first glance (but it's not).